modules/ROOT/pages/async-scope-reference.adoc

```diff
@@ -14,6 +14,21 @@ image::async-scope-schematic.png[Async+scope+schematic]
 
 Because the Async scope is executed in a "fire and forget" manner, the result of the processing within the scope is not available in the main flow.
 
+== Async Configuration
+
+Async scopes are configurable.
+
+[%header,cols="1,4"]
+|===
+| Field | Description
+| Display Name (`name`) | Name for the Async scope. 
+| Max Concurrency (`maxConcurrency`) a| Optional. Sets the maximum number of concurrent messages that the scope can process. By default, the container thread pool determines the maximum number of threads to use to optimize the performance when processing messages. When the flow is processing the maximum number of concurrent messages, it cannot receive additional requests.
+
+Set `maxConcurrency` to `1` to cause the scope to process requests one at a time.
+
+See xref:execution-engine.adoc#backpressure[Back-pressure] for details about Mule's behavior after the the maximum concurrency value is reached.
```

Comment:
```suggestion
See xref:execution-engine.adoc#backpressure[Back-Pressure Management] for details about Mule's behavior after reaching the maximum concurrency value.
```

Reply:
Should it be "Mule" rather than "Mule's"? I was taught not to anthropomorphize product names.

Reply:
incorporated outside of suggestion to avoid a soddy SoD